src/NzbDrone.Core/Notifications/MediaBrowser/MediaBrowser.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+using FluentValidation.Results;
+using NzbDrone.Common;
+using NzbDrone.Common.Extensions;
+using NzbDrone.Core.Tv;
+
+namespace NzbDrone.Core.Notifications.MediaBrowser
+{
+    public class MediaBrowser : NotificationBase<MediaBrowserSettings>
```

Comment:
Fair enough. This requires some other architectural changes, which I agree is a good idea (added to Trello: https://trello.com/c/2AW7y73n/912-be-more-explicit-with-class-names-for-3rd-party-integrations).

I'd like to get this merged in and we can make those other changes in another PR.